### ISSUE TITLE
Inaccurate keyframe interval, fps metrics when querying stream info over API

### ISSUE THREAD

**Issue by hernanrz** [AirenSoft/OvenMediaEngine#1445]
**Describe the bug**
When querying a stream info (api endpoint: `v1/vhosts/default/apps/live/streams/<stream>`), the server reports strange values for keyframe interval and fps (fractional values)

![image](https://github.com/AirenSoft/OvenMediaEngine/assets/13426225/4a69a564-2d0b-4d36-9096-c4132282dbac)
while querying the same stream but in the origin server returns correct FPS values, although the keyframe value reported is definitely different from the one set in OBS

![image](https://github.com/AirenSoft/OvenMediaEngine/assets/13426225/34fbad44-050d-46e8-af41-099ff56013ce)

OME origin/edge Config https://gist.github.com/hernanrz/87fb0ff693cbd91ff50c3f6be1436a58

Version: OvenMediaEngine v0.16.0 ((From archive)) running with docker image airensoft/ovenmediaengine@sha256:167de56e8b6600e0f459038780fff17659d7f835a3464834c20403b1e694564e

**Comment by getroot** [comment]
This is the difference between the config value and the measured value. In the API, the origin responds with setting values, and the edge responds with actual measured values.
To eliminate this confusion, I will output config values and measured values separately in the API response.

**Comment by hernanrz** [comment]
That sounds like a reasonable solution, thank you 👍

**Comment by getroot** [comment]
@hernanrz 

I added the following information to the GET Stream API: It has been committed to the master branch and will be available from the next release.

bitrate_conf
bitrate_measured
framerate_conf
framerate_measured
keyFrameIntervalConf
keyFrameIntervalMeasured

Sorry I'm so late!

**Comment by getroot** [comment]
I updated this to this:

“bitrate_latest” is the value measured in the most recent second.
“framerate_latest” is the value measured in the most recent second.
"keyFrameInterval_latest" is the number of delta frames between the immediately previous keyframe and the keyframe before it.

"_avg" is the overall average.

"bitrate_avg": "997606",
"bitrate_conf": "1000000",
"bitrate_latest": "874306",
"framerate_avg": 30.008920669555664,
"framerate_conf": 30.0;
"framerate_latest": 29.693487167358398,
"keyFrameInterval_avg": 29,
"keyFrameInterval_conf": 30,
"keyFrameInterval_latest": 30,

**Comment by naanlizard** [comment]
I was going to suggest not changing the old names, this is a good update thank you!

**Comment by getroot** [comment]
@bchah I thought you said you needed this too, right?

**Comment by bchah** [comment]
@getroot yes, thank you! I saw the commits and am excited to incorporate this in the next release. It gives users a way to see their "current" frame and bit rates. Great work and thank you very much :)
